evaluations.css

```diff
@@ -0,0 +1,97 @@
+		body {
+			min-width: 100%;
```

Comment:
It's due to a little thing we, developers, like to call _overthinking_. :smile: We've all been there (heck, why lie? we're still there a **lot** of the time!), but that's precisely where the value of code reviews lies - in allowing people who aren't as intimately familiar with the code (and thus often viewing it from a subjective standpoint) to poke around, ask questions, and hopefully catch stuff that's not thought through thoroughly enough (whoa, talk about a tongue twister!) or, quite the contrary, over-engineered.